src/commands/HelpCommand.ts

```diff
@@ -17,7 +17,7 @@ export class HelpCommand extends BaseCommand {
         if (command && !command.meta.disable) {
             message.channel.send(
                 new MessageEmbed()
-                    .setTitle(`Help for ${command.meta.name} command`)
+                    .setTitle(`Information for ${command.meta.name} command`)
```

Comment:
```suggestion
                    .setTitle(`Information for the ${command.meta.name} command`)
```
Add `the` before the command name